webui/react/src/pages/ExperimentList.tsx

```diff
@@ -124,10 +124,14 @@ const ExperimentList: React.FC = () => {
   }, [ selectedExperiments ]);
 
   const fetchExperiments = useCallback(async (): Promise<void> => {
-    const response = await getExperimentList(sorter, pagination, filters, search);
-    const experiments = decodeExperimentList(response.experiments || []);
-    setTotal(response.pagination?.total || 0);
-    setExperiments(experiments);
+    try {
+      const response = await getExperimentList(sorter, pagination, filters, search);
```

Comment:
nonblocking: as is this api call isn't going through apibuilder meaning it's not using any of the surrounding work that it does. we might want to either use that or build one more tailored to autogenerated API bindings.

this might be better left for future.

Reply:
100% agree, there will be enough of a pattern like this one that it will be good to refactor cleanly